### ISSUE TITLE
ImportOrderChecker did not test group ordering

### ISSUE THREAD

**Issue by espertus** [scalastyle/scalastyle#317]
My configuration file includes [the example configuration for ImportOrderChecker(http://www.scalastyle.org/rules-dev.html#org_scalastyle_scalariform_ImportOrderChecker):

```
<check enabled="true" class="org.scalastyle.scalariform.ImportOrderChecker" level="warning">
 <parameters>
  <parameter name="groups">java,scala,others</parameter>
  <parameter name="group.java">javax?\..+</parameter>
  <parameter name="group.scala">scala\..+</parameter>
  <parameter name="group.others">.+</parameter>
 </parameters>
</check>
```

Nonetheless, there were no warnings for a file with this order of imports:
```
import cascading.property.AppProps
import cascading.tuple.Fields
import java.util.Properties
```

I expected a warning because the line beginning  `import java` did not precede the other imports.
